examples/rbd/storageclass.yaml

```diff
@@ -53,14 +53,22 @@ parameters:
    # https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options
    # For nbd options refer
    # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
-   # mapOptions: lock_on_read,queue_depth=1024
+   # Format:
+   # mapOptions: "<mounter1>:op1,op2;<mounter2>:op1,op2"
```

Comment:
```suggestion
# mounter represents the mounter it can be either krbd and nbd.
op1,op2 represent the mounter specific operation.
#mapOptions: "<mounter>:op1,op2;<mounter>:op1,op2"
```
instead of mounter1 and mounter to mention

Reply:
@Madhu-1 could you please clarify this a bit more.

Reply:
Do you mean fix typo ?
`s/mounter/mounter2/`

Reply:
or change this to
`#mapOptions: "krbd:op1,op2;nbd:op1,op2"`

Reply:
sorry ignore, I got it